### ISSUE TITLE
calling eslint index.js does nothing

### ISSUE THREAD

**Issue by jonkemp** [eslint/eslint#4550]
I'm having the same issue as #2388 but that issue is closed. Running `eslint index.js` returns nothing.

```bash
$ eslint -v
v1.10.1
```

```bash
$ eslint index.js --debug
  eslint:cli Running on files +0ms
  eslint:glob-util Creating list of files to process. +49ms
  eslint:cli-engine Processing /Library/WebServer/Documents/gulp-useref/index.js +4ms
  eslint:cli-engine Linting /Library/WebServer/Documents/gulp-useref/index.js +1ms
  eslint:config Constructing config for /Library/WebServer/Documents/gulp-useref/index.js +1ms
  eslint:config Using .eslintrc and package.json files +0ms
  eslint:config Loading /Library/WebServer/Documents/gulp-useref/package.json +3ms
  eslint:config-file Loading package.json config file: /Library/WebServer/Documents/gulp-useref/package.json +0ms
  eslint:config Merging command line environment settings +2ms
  eslint:cli-engine Linting complete in: 110ms +96ms
```

**Comment by eslintbot** [comment]
@jonkemp Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by michaelficarra** [comment]
Congratulations.

**Comment by platinumazure** [comment]
You have no linting errors. That's a good thing! Or are you expecting lint
errors? If so, please follow the request given by the ESLint bot and show
us your file and configuration.

**Comment by jonkemp** [comment]
Ah, thanks. Sorry.
